test/rng_testsuite/extra_methods_tests/distributions_methods.pass.cpp

```diff
@@ -0,0 +1,269 @@
+// -*- C++ -*-
+//===-- engines_methods.hpp ----------------------             -------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+//
+// Abstract:
+//
+// Testing of different distributions' methods
+
+#include "support/utils.h"
+#include <iostream>
+
+#if _ONEDPL_BACKEND_SYCL
+#    include <cmath>
+#    include <vector>
+#    include <CL/sycl.hpp>
+#    include <oneapi/dpl/random>
+
+#    define SEED 777
+#    define N_GEN 1000
+
+template <class T>
+std::int32_t
+check_params(oneapi::dpl::uniform_int_distribution<T>& distr,
+             typename oneapi::dpl::internal::type_traits_t<T>::element_type a,

```

Comment:
I believe there can be an alias to reduce number of symbols, e.g. 
```cpp
template<typename T>
using Element_type = typename oneapi::dpl::internal::type_traits_t<T>::element_type
```

Reply:
changed to proposed approach